docs/source/losses.rst

```diff
@@ -39,16 +39,12 @@ DQN Loss
 .. autofunction:: pl_bolts.losses.rl.dqn_loss
     :noindex:
 
----------------
```

Comment:
I think that @williamFalcon  made it on purpose, so is there another way how to have a horizontal line?

Reply:
I looked through the docs, but I'm still not sure if there's another way to draw a horizontal line...
- [reStructuredText Markup Specification](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html)
- [Sphinx documentation contents](https://www.sphinx-doc.org/en/master/contents.html)

Reply:
could it be this one? https://docutils.sourceforge.io/docs/ref/doctree.html#transition
so let's just remove it at this moment :]